hack/ctrcheck

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+usage() {
+    echo "usage: ctrcheck docker ps [FILTER] [TIMEOUT]"
+    echo "example: ctrcheck docker ps --filter "name=jdoe" 60"
```

Comment:
@neha-viswanathan When we chatted about this, the flexibility of supplying a filter could be nice, but I was just thinking something simple, more along the lines of `ctrcheck NAME [STATUS] [TIMEOUT]`, where status could default to `running` and timeout could default to `300` (5m) (we can tune the value based on travis performance).